README.md

```diff
@@ -140,6 +140,14 @@ torus network params
 torus misc circulating-supply
 ```
 
+### Acquiring Tokens from a Testnet Faucet
+
+```sh
+# This command will send 15 TOR to the key-name address.
+# This command is only available on testnet.
+torus --testnet balance run-faucet key-name
```

Comment:
kinda nitpicky: but using `<key-name>` instead of key-name makes it clear that users should replace it with their actual key name, following common documentation conventions for placeholders.

```suggestion
torus --testnet balance run-faucet <key-name>
```